apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/internal/flows.kt

```diff
@@ -0,0 +1,49 @@
+package com.apollographql.apollo3.internal
+
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.FlowCollector
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flow
+
+/**
+ * This is copied from [Flow], which is marked [ExperimentalCoroutinesApi].
```

Comment:
Same as for `ChannelWrapper`, we should link the source for attribution + remember where we got it from in 6 months.

Reply:
Good point!